elastic/README.md

```diff
@@ -1,32 +1,86 @@
-# Elastic Integration
+# Elasticsearch Integration
 
-## Overview
+# Overview
 
-Get metrics from elastic service in real time to:
+Stay up-to-date on the health of your Elasticsearch cluster, from its overall status down to JVM heap usage and everything in between. Get notified when you need to revive a replica, add capacity to the cluster, or otherwise tweak its configuration. After doing so, track how your cluster metrics respond.
 
-* Visualize and monitor elastic states
-* Be notified about elastic failovers and events.
+The Datadog Agent's Elasticsearch check collects metrics for search and indexing performance, memory usage and garbage collection, node availability, shard statistics, disk space and performance, pending tasks, and many more. The Agent also sends events and service checks for the overall status of your cluster.
 
-## Installation
+# Installation
 
-Install the `dd-check-elastic` package manually or with your favorite configuration manager
+The Elasticsearch check is packaged with the Datadog Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Elasticsearch nodes, or on some other server if you use a hosted Elasticsearch (e.g. Elastic Cloud). If you need the newest version of the Elasticsearch check, install the `dd-check-elastic` package; this package's check will override the one packaged with the Agent. See the [integrations-core](https://github.com/DataDog/integrations-core#installing-the-integrations) repository for more details.
```

Comment:
True, though I don't think most READMEs are exemplary right now; they are auto-generated stubs. I'm going through each (slowly but surely) to update, so what we have in place isn't necessarily how docs ought to look. :)